crates/swc_ecma_transforms_base/src/hygiene/tests.rs

```diff
@@ -313,8 +317,8 @@ fn for_loop() {
         },
         "
             for(var a=1;;) {}
-            for(var a1 of foo) {}
-            for(var a2 = 3;;) {}
+            for(var a of foo) {}
```

Comment:
One of the main idea of this PR is to leave the user input `vars` untouched.
But when updating this test file, I found many cases where the `var` was intentionally changed.
Maybe I'm thinking in the wrong way.

Reply:
Sometimes it's inevitable, as `var` and `let`/`const` have different scoping.

Reply:
If there are some cases where `var` ident of different scopes are together and conflict with each other, this pull request is meaningless and I will close it.

Reply:
Those updates are mostly not intentional.
The renamed `var`s in tests are renamed just because it works and it was required to pass the test.

Reply:
Oh, almost all renamings in his files are intentional.
They have different syntax contexts, so those are different variables.

Reply:
Tests in this file are extracted from other tests, and those are tests for injected identifiers.